packages/configs/typescript/tsconfig.vue.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    // required in `vue` projects
+    "jsx": "preserve",
+    // see: https://github.com/vuejs/test-utils/issues/1538
+    "skipLibCheck": true,
+    "types": ["vite/client"]
```

Comment:
`jsx`, `skipLibCheck` and `types` are Vue specific. The rest of the properties in the current `ui-vue` [_tsconfig.json_](https://github.com/aws-amplify/amplify-ui/blob/main/packages/vue/tsconfig.json) can be extended from the shared base [_tsconfig.base.json_](https://github.com/aws-amplify/amplify-ui/blob/main/packages/configs/typescript/tsconfig.base.json) or were extraneous